src/unleash.ts

```diff
@@ -84,9 +93,21 @@ export class Unleash extends EventEmitter {
     bootstrapOverride,
     storageProvider,
     disableAutoStart = false,
+    skipInstanceCountWarning = false,
   }: UnleashConfig) {
     super();
 
+    Unleash.instanceCount++;
+
+    this.on(UnleashEvents.Error, (err) => {
+      console.error(err?.message);
+    });
+
+    if(!skipInstanceCountWarning && Unleash.instanceCount > 10) {
+      const error = new Error('The unleash SDK has been initialized more than 10 times');
```

Comment:
need to wrap this in a process.nextTick() to ensure using code have a chance to register an event listener before it triggers.